### PR TITLE
[13.0][MIG] res_company_active

### PR DESCRIPTION
**Pull request by quentinDupont**
Migration from 12.0

This module adds `active` fields on company model.

res_company_form

A control is realized, to check if the disabled company is the current company for active users.

validation_error

### CODE REVIEW

File: res_company_active/demo/res_company.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright (C) 2019 - Today: GRAP (http://www.grap.coop)
+@author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+<odoo>
```
Comment (florian-dacosta) [inline res_company_active/demo/res_company.xml:7]: Usually demo data are with noupdate="1", isn't it better?
Not blocking from my part though.

**Review by florian-dacosta — APPROVED** [review]

File: res_company_active/demo/res_company.xml (hunk 1 above)
Comment (quentinDupont) [reply res_company_active/demo/res_company.xml:7]: I agree with that and I made the change, thanks !

**Comment by HaraldPanten** [comment]
Could you rebase?

**Comment by quentinDupont** [comment]
> Could you rebase?

Done :slightly_smiling_face:

**Comment by HaraldPanten** [comment]
@quentinDupont you should check pre-commit

**Comment by HaraldPanten** [comment]
Hi @quentinDupont normally you should have pre-commit (black, isort, prettier) in a separate commit, I mean:

1- [IMP] black, isort, prettier.
2- Migration commit

Do you know what I mean?

**Comment by quentinDupont** [comment]
> Hi @quentinDupont normally you should have pre-commit (black, isort, prettier) in a separate commit, I mean:
> 
> 1- [IMP] black, isort, prettier.
> 2- Migration commit
> 
> Do you know what I mean?

Oh ok I didn't know so I squashed these commits.
Is it mandatory ? Cause I don't know how to unsquash these.

Also 1st precommit fails with "This check failed" an no more information :man_shrugging: 
And Runbot fails but I cant access logs..

**Pull request closed without merging**